picker/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js" defer></script>
+
+	<title>Goots Gallery</title>
+	<meta name="description" content="The ultimate goots viewing experience.">
+</head>
+<body>
+<main>
+
+<!--
+Re-generate by running the following in the goots/ directory and pasting the result in here:
+ls | sed -ne 's|\(.*\)\.\(.*\)|<img src="../goots/\1.\2" alt=":\1:" title=":\1:" height="128" width="128">|p' | pbcopy
+-->
```

Comment:
I'll probably automate updating this as part of `build.py` once https://github.com/christrotter/gootsmoji/pull/20 is merged.